content/en/docs/tasks/access-api-from-pod/access-api-from-pod.md

```diff
@@ -0,0 +1,113 @@
+---
+title: Accessing the Kubernetes API from a Pod
+reviewers:
+- sftim
+description: Accessing the Kubernetes REST API from within a Pod
+content_type: task
+---
+
+<!-- overview -->
+
+This guide demonstrates how to access the Kubernetes API from within a pod.
+
+## {{% heading "prerequisites" %}}
+
+{{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
```

Comment:
nit - all Kubernetes versions support Pods and the cluster's API, so we don't need to check compatibility.
```suggestion
{{< include "task-tutorial-prereqs.md" >}}
```

Reply:
Done